src/System.ComponentModel.Primitives/src/System/ComponentModel/DescriptionAttribute.cs

```diff
@@ -0,0 +1,89 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel
+{
+    /// <devdoc>
+    ///    <para>Specifies a description for a property
+    ///       or event.</para>
+    /// </devdoc>
+    [AttributeUsage(AttributeTargets.All)]
+    public class DescriptionAttribute : Attribute, IIsDefaultAttribute
+    {
+        /// <devdoc>
+        /// <para>Specifies the default value for the <see cref='System.ComponentModel.DescriptionAttribute'/> , which is an
+        ///    empty string (""). This <see langword='static'/> field is read-only.</para>
+        /// </devdoc>
+        public static readonly DescriptionAttribute Default = new DescriptionAttribute();
+        private string _description;
+
+        /// <devdoc>
+        ///    <para>[To be supplied.]</para>
+        /// </devdoc>
+        public DescriptionAttribute() : this(string.Empty)
+        {
+        }
+
+        /// <devdoc>
+        ///    <para>Initializes a new instance of the <see cref='System.ComponentModel.DescriptionAttribute'/> class.</para>
+        /// </devdoc>
+        public DescriptionAttribute(string description)
+        {
+            _description = description;
+        }
+
+        /// <devdoc>
+        ///    <para>Gets the description stored in this attribute.</para>
+        /// </devdoc>
+        public virtual string Description
+        {
+            get
+            {
+                return DescriptionValue;
+            }
+        }
+
+        /// <devdoc>
+        ///     Read/Write property that directly modifies the string stored
+        ///     in the description attribute. The default implementation
+        ///     of the Description property simply returns this value.
+        /// </devdoc>
+        protected string DescriptionValue
+        {
+            get
+            {
+                return _description;
+            }
+            set
+            {
+                _description = value;
+            }
+        }
```

Comment:
Nit: since _description is only used here, you could simplify this by getting rid of the explicit static field and just doing:
```C#
protected string DescriptionValue { get; set; }
```